artist/field/actuator_array.py

```diff
@@ -40,6 +44,8 @@ def __init__(self, actuator_list_config: ActuatorListConfig) -> None:
         ----------
         actuator_list_config : ActuatorListConfig
             The configuration parameters for the actuators.
+        device : torch.device
+            The device on which to initialize tensors (default is CPU).
         """
         super().__init__()
```

Comment:
```suggestion
        super().__init__()
        device = torch.device(device)
```
If we allow passing a string for the `device`, it should be converted to a `torch.device` at the beginning of each function / method.